com.github.sdbg.debug.ui/src/com/github/sdbg/ui/launchers/IChromeLaunchShortcutExt.java

```diff
@@ -0,0 +1,11 @@
+package com.github.sdbg.ui.launchers;
+
+import org.eclipse.core.resources.IResource;
+
+public interface IChromeLaunchShortcutExt {
```

Comment:
I'm after launching from the urls and if something doesn't exist create a launcher. I didn't want to extend the launching system in replace of something custom... 

![61682cde-f18f-11e4-9cae-1e1265886a90](https://cloud.githubusercontent.com/assets/1326504/7477204/26a15a70-f307-11e4-832e-bdea7107bd52.png)